wiki/illuminations/index.md

```diff
@@ -336,29 +336,33 @@ The Biome Settings allow you to customize the **Firefly Spawn Rate**, **Firefly
 
 #### Can I include this mod in a modpack?
 
-**Yes**: You can. Go ahead, don't bother asking. Please however provide credit and a link to either the GitHub repository or Curse Forge project page.
+**Yes**: You can. Go ahead, don't bother asking. Please however provide credit and a link to either the [GitHub repository](https://github.com/ladysnake/illuminations/) or [Curse Forge project page](https://www.curseforge.com/minecraft/mc-mods/illuminations).
+
+#### Can you port this mod to forge? Can we get a version XY?
+
+Sorry, we **don't** allow backports or ports to other modloaders for this mod.
 
 #### Can I use this mod on a vanilla server?
 
 **Yes**: If you have installed this mod on your Minecraft client, you will be able to connect to vanilla Minecraft servers and the ambient illuminations will work as intended.
 
 #### Are auras visible to other players?
 
-**Yes**: If the player has the mod installed on their client, your aura will be visible to them, just like their aura will be visible to you if they possess one.
+**Yes**: If the player has Illuminations installed on their client, your aura will be visible to them, just like their aura will be visible to you if they possess one.
 
-#### How to get an aura?
+#### How to get cosmetics?
 
-**Auras are currently only available to donators**, if they have donated more than 1$.
+Cosmetics are available to donators who donated at least 2$ on [ko-fi](https://ko-fi.com/doctor4t/shop) for tier 1 (Twilight Aura) and 7$ for tier 2.
```

Comment:
Dollars or Euros? Also I believe that some double-checking should occur on the currency style. If in dollars, it may be useful to also specify which (probably USD but could be others)

Reply:
Euros, idk why it said $ tbh, also that seems to generally be outdated now, old commit